secure-area-test-app/src/main/java/com/android/identity/secure_area_test_app/MainActivity.kt

```diff
@@ -156,13 +157,15 @@ class MainActivity :  FragmentActivity() {
             listOf(Crypto.createX509v3Certificate(
                 softwareAttestationKey.publicKey,
                 softwareAttestationKey,
+                null,
                 Algorithm.ES256,
                 "1",
                 "CN=Software Attestation Root",
                 "CN=Software Attestation Root",
                 validFrom,
                 validUntil,
-                emptyList<Crypto.X509v3Extension>()
+                emptySet(),
+                emptyList()
```

Comment:
common practice in kotlin would be `setOf()`and `listOf()` (which run exactly `emptySet()` and `emptyList()`)

Reply:
Done.